ols/utils/token_handler.py

```diff
@@ -125,7 +125,7 @@ def truncate_rag_context(
 
         for node in retrieved_nodes:
 
-            score = float(node.score)
+            score = float(node.get_score())
```

Comment:
Maybe `get_score(raise_error=False)` to ensure `0` is returned instead of `None`.

Reply:
hmm do we need to put the error under the carpet? but can do it

Reply:
updated @onmete

Reply:
Just thinking it may be safer due to changes in llm libraries in general. :)